src/main/java/uk/gov/hmcts/reform/civil/helpers/LocationHelper.java

```diff
@@ -183,7 +182,6 @@ private Optional<RequestedCourt> getSpecClaimantRequestedCourt(CaseData caseData
     private Optional<RequestedCourt> getUnspecClaimantRequestedCourt(CaseData caseData) {
         return Optional.ofNullable(caseData.getCourtLocation())
             .map(courtLocation -> RequestedCourt.builder()
-                .requestHearingAtSpecificCourt(YesOrNo.YES)
```

Comment:
was this removed for a reason?

Reply:
yes requestHearingAtSpecificCourt is deprecated